security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,37 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-4-0"]
+== OpenShift Compliance Operator 1.4.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.4.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:7658[RHBA-2023:7658 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Users can now pause scan schedules by setting the `ScanSetting.suspend` attribute to `True`. This allows users to suspend a scan schedule and reactivate it without the need to delete and re-create the `ScanSettingBinding`. This simplifies pausing scan schedules during maintenance periods. link:https://issues.redhat.com/browse/CMP-2123[CMP-2123]
```

Comment:
One final nit with the PR @GroceryBoyJr  - be sure to put all links in the enhancements and bugs in **BOLD**, and the entire `link:` statement goes in parenthesis: 

`(link:https://issues.redhat.com/browse/CMP-2123[*CMP-2123*])`

See more examples in CO 1.1.0, around line 200 or so on this file.

Reply:
@sheriff-rh This is done, thank you!